emptytheme/assets/alignments-front.css

```diff
@@ -0,0 +1,79 @@
+/*
+ * Alignments, loaded in the front-end only.
+ */
+
+body {
+	margin: 0;
+}
+
+* {
+	box-sizing: border-box;
+}
+
+.wp-site-blocks,
+.wp-block-template-part.alignfull {
+	padding: 0 var(--wp--custom--margin--horizontal);
+}
+
+.wp-site-blocks > *:not(.wp-block-post-content),
+.wp-site-blocks .wp-block-post-content > * {
+	max-width: var(--wp--custom--width--default);
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.wp-site-blocks .alignwide {
+	width: var(--wp--custom--width--wide);
+	max-width: 100%;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.wp-site-blocks .alignfull {
+	transform: translateX(calc(0px - var(--wp--custom--width-horizontal)));
+	width: calc(100% + (2 * var(--wp--custom--width-horizontal)));
+	max-width: calc(100% + (2 * var(--wp--custom--width-horizontal)));
```

Comment:
Where is this defined? Should it be `--wp--custom--margin-horizontal`?

Firefox devtools tells me it is not set:

<img width="591" alt="Screenshot 2020-11-18 at 13 27 46" src="https://user-images.githubusercontent.com/3593343/99530869-2bba4300-29a2-11eb-9746-41e3301945c0.png">

Reply:
Those are generated from the theme.json declaration (see docs for [free-from custom properties](https://developer.wordpress.org/block-editor/developers/themes/theme-json/#free-form-css-custom-properties)). From what I see in [the file](https://github.com/WordPress/theme-experiments/pull/81/files#diff-fbd7c92cbefc0f9735d61095ae8e3b5a5b42521a1853477566cd54d981d7ba1aR16), the ones that will be generated are: `--wp--custom--width--default`, `--wp--custom--width--wide`, and `--wp--custom--margin--horizontal`.

Reply:
Good catch! That should be `--wp--custom--margin--horizontal`. I've updated that in 5a32027